runtime/src/invoke_dpe.rs

```diff
@@ -56,4 +83,8 @@ impl InvokeDpeCmd {
             Err(CaliptraError::RUNTIME_INSUFFICIENT_MEMORY)
         }
     }
+
+    fn is_caller_p1(pl0_pauser: u32, flags: u32, locality: u32) -> bool {
+        flags & 1 == 0 && locality != pl0_pauser
```

Comment:
nit: Can you defined a const for this? `const PL0_PAUSER_FLAG: u32 = 1`. Alternatively you could use bitflags, although that might be a little overkill since there's only one flag.

Reply:
Done